app/routes/assets/asset.py

```diff
@@ -242,10 +255,111 @@ async def get_stats(asset_id: UUID = Path(...)):
     "/{asset_id}/fields",
     response_class=ORJSONResponse,
     tags=["Assets"],
-    response_model=FieldMetadataResponse,
+    response_model=FieldsMetadataResponse,
 )
 async def get_fields(asset_id: UUID = Path(...)):
-    asset: ORMAsset = await assets.get_asset(asset_id)
-    fields: List[FieldMetadata] = [FieldMetadata(**field) for field in asset.fields]
+    asset_metadata = await metadata_crud.get_asset_metadata(asset_id)
+    fields: List[ORMFieldMetadata] = await metadata_crud.get_asset_fields(
+        asset_metadata.id
+    )
+
+    return FieldsMetadataResponse(data=fields)
+
+
+@router.patch(
```

Comment:
I don't think it's super necessary, but it feels weird you can PATCH a specific field, but you can't do a GET on a specific field (only all fields). Should we include it for consistency?

Reply:
Good catch, I'm actually leaning to modifying the PATCH to update any number of fields.